Patch to enable "hidden" clock in the Clock app shipped
with WebOS 1.0.4.  Also modded to show day and date.

Mod by frankos72 @ precentral and Phrozen
Patch submitted by JackieRipper
Index: /usr/palm/applications/com.palm.app.clock/themes/manualanalog/manualanalog-clock-functions.js
===================================================================
--- .orig/usr/palm/applications/com.palm.app.clock/themes/manualanalog/manualanalog-clock-functions.js
+++ /usr/palm/applications/com.palm.app.clock/themes/manualanalog/manualanalog-clock-functions.js
@@ -89,8 +89,8 @@
 			}
 			// format date
 		
-			//this.controller.get('date').textContent = Mojo.Format.formatDate(now, {"date":"short"})
-			//this.controller.get('day').textContent = Mojo.Format.getDateTimeHash().medium.day[now.getDay()];
+			this.controller.get('date').textContent = Mojo.Format.formatDate(now, {"date":"short"})
+			this.controller.get('day').textContent = Mojo.Format.getDateTimeHash().medium.day[now.getDay()];
 	}
 
 	ManualanalogClockFunctions.prototype.alarm = function() {
Index: /usr/palm/applications/com.palm.app.clock/themes/manualanalog/manualanalog-clock.html
===================================================================
--- .orig/usr/palm/applications/com.palm.app.clock/themes/manualanalog/manualanalog-clock.html
+++ /usr/palm/applications/com.palm.app.clock/themes/manualanalog/manualanalog-clock.html
@@ -1,8 +1,8 @@
 
 <div class="clock-container">
 	<div id="hands">
-		   <!-- <div id="day" class="day"></div>
-		   <div id="date" class="date"></div> -->
+		   <div id="day" class="day"></div>
+		   <div id="date" class="date"></div>
 
 		   <div id="hour"></div>
 		   <div id="minute"></div>
Index: /usr/palm/applications/com.palm.app.clock/themes/themes.json
===================================================================
--- .orig/usr/palm/applications/com.palm.app.clock/themes/themes.json
+++ /usr/palm/applications/com.palm.app.clock/themes/themes.json
@@ -10,5 +10,10 @@
 		"nicename":"Analog",
 		"description":"",
 		"source": "themes/analog/"
+	},
+		"name":"manualanalog",
+		"nicename":"Analog Black",
+		"description":"",
+		"source": "themes/manualanalog/"
 	}
-]
\ No newline at end of file
+]
